tgui/packages/tgui/interfaces/Keybinds/Keybind.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+import { Button, LabeledList } from 'tgui-core/components';
+import { isEscape } from 'tgui-core/keys';
+import { BooleanLike } from 'tgui-core/react';
+
+import { useBackend } from '../../backend';
+import { formatKeyboardEvent, isStandardKey } from './formatKeyboardEvent';
+import { KeybindData, KeybindsData } from './types';
+
+interface KeybindProps {
+  keybind: KeybindData;
+  isFocused: BooleanLike;
+  setFocusedKey: (string) => void;
+}
+export const Keybind = (props: KeybindProps) => {
+  const { act } = useBackend<KeybindsData>();
+  const { keybind, isFocused, setFocusedKey } = props;
+  return (
+    <LabeledList.Item label={keybind.label}>
+      <Button
```

Comment:
Should these be children, or the `buttons` prop of the `LabeledList.Item` above?

Reply:
I think in this case, the button is the content/focus of the list, so in makes sense to be children. As opposed to elsewhere, where most of the time, the button serves to do an action on the list's content.